### PR TITLE
Update closures.md to correct a typo

### PR DESCRIPTION
**Pull request by sinansonmez**

### CODE REVIEW

File: src/fn/closures.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ variable names *must* be specified.
 
 Other characteristics of closures include:
 * using `||` instead of `()` around input variables.
-* optional body delimination (`{}`) for a single expression (mandatory otherwise).
+* optional body delimination (`{}`) for a single line expression (mandatory otherwise).
```
Comment (kberov) [inline src/fn/closures.md:17]:
```suggestion
* optional body delimitation (`{}`) for a single line expression (mandatory otherwise).
```

**Comment by marioidival** [comment]
@kberov can you fix the conflicts? thank you!

**Comment by kberov** [comment]
Unfortunately I do not know how can I do that. I see only that @sinansonmez accepted my proposal into his PR.

**Comment by kberov** [comment]
@sinansonmez, can you please resolve the conflict and push back to the PR. I cannot do it.

**Review by marioidival — APPROVED** [review]

**Pull request merged**